Packager/CMakeLists.txt

```diff
@@ -19,8 +19,6 @@ set(PLUGIN_NAME Packager)
 set(MODULE_NAME WPEFramework${PLUGIN_NAME})
 
 find_package(${NAMESPACE}Plugins REQUIRED)
-find_package(libprovision REQUIRED)
-find_package(LibOPKG REQUIRED)
```

Comment:
why do you remove LibOPKG ? that library is required for building the packager plugin,no?

Reply:
Yes its required but we don't have a cmake config file for opkg currently. LDFLAGS for opkg is passed from recipe. I can remove REQUIRED if you want so that it doesn't stop the build.